### DISCUSSION TITLE
support spatial analysis - Standard for Geographic information

### DISCUSSION THREAD

**Discussion by ElCep** [cormas/cormas#780] [Ideas]
The OGC has written standards for the implementation of space operators. Following "[OpenGIS ® Implementation Standard for Geographic information - Simple feature access](https://www.ogc.org/standards/sfa)" it will be great to it have in CORMAS

Methods that support spatial analysis (6.1.2.4 p. 17) : 
* Distance : Returns the shortest distance between any two Points
* Buffer : Returns a geometric object that represents all Points whose distance from this geometric object is less than or equal to distance
* ConvexHull : Returns a geometric object that represents the convex hull of this geometric object
* Intersection : Returns a geometric object that represents the Point set intersection of this geometric object with another Geometry.
* Union : Returns a geometric object that represents the Point set union of this geometric object with anotherGeometry
* Difference : Returns a geometric object that represents the Point set difference of this geometric object with anotherGeometry
* SymDifference : Returns a geometric object that represents the Point set symmetric difference of this geometric object with anotherGeometry

**Comment by SergeStinckwich** [comment]
Ok same reply than in #139
